common/djangoapps/student/models.py

```diff
@@ -1361,6 +1379,21 @@ def update_enrollment(self, mode=None, is_active=None, skip_refund=False):
                 self.send_signal(EnrollStatusChange.unenroll)
 
         if mode_changed:
+            if COURSEWARE_PROCTORING_IMPROVEMENTS.is_enabled(self.course_id):
+                # If mode changed to one that requires proctoring, send proctoring requirements email
+                course_module = modulestore().get_course(self.course.id)
+                proctored_exams_enabled = course_module.enable_proctored_exams
+                if self.has_proctoring_requirements(proctored_exams_enabled):
+                    # Humanize proctoring provider string
+                    proctoring_provider = capwords(course_module.proctoring_provider.replace('_', ' '))
+                    id_verification_url = IDVerificationService.get_verify_location()
+                    email_context = {'user': self.user, 'course_name': self.course.display_name,
+                                     'proctoring_provider': proctoring_provider,
+                                     'proctoring_requirements_url': ('https://support.edx.org/hc/en-us/articles/'
```

Comment:
I think we need to move this URL to a settings variable so that it's not enabled by default on open edX instances.